charts/schema-registry/README.md

```diff
@@ -0,0 +1,71 @@
+<!--
+
+  Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+-->
+
+# Schema Registry Helm Chart
+
+Installs [Pravega Schema Registry](https://github.com/pravega/schema-registry) atop Kubernetes.
+
+## Introduction
+
+This chart bootstraps a [schema registry](https://github.com/pravega/schema-registry) deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+
+## Prerequisites
+  - Kubernetes 1.15+
+  - Helm 3+
+
+## Installing the Chart
+
+To install the chart with the release name `my-release`:
+
+```
+$ helm install my-release charts/schema-registry
+```
+
+This command deploys schema registry on the Kubernetes cluster with default configuration. The [configuration](#configuration) section lists the parameters that can be configured during installation.
+
+## Uninstalling the Chart
+
+To uninstall/delete the `my-release` deployment:
+
+```
+$ helm uninstall my-release
+```
+
+This command removes all the Kubernetes components associated with the chart and deletes the release.
+
+## Configuration
+
+The following table lists the configurable parameters of the schema registry chart and their default values.
+
+| Parameter | Description | Default |
```

Comment:
A few of these config items do not follow the [new config naming scheme](https://github.com/pravega/pravega/wiki/Configuration-Naming-Guidelines). 

* `replicas` -> `replica.count` 
* `controllerUri` -> `controller.connect.uri` (Connection params to other systems have <target>.connect...)
* `storeType` -> `store.type` 
* `serviceAccount.create` -> `serviceAccount.create.enable` (booleans must have enable)
* `authentication.enabled` -> `security.auth.enable` (Because auth represents both authentication and authorization)
* `authentication.passwordAuthSecret` - What is this? 
* `authentication.userPasswordFile` -> `security.auth.pwdAuthHandler.accountsDb.location`
* `authentication.tokenSigningKey` -> `security.auth.delegationToken.signingKey.basis`
* `tls.enabled` -> `security.tls.enable`
* `tls.secret` - What is this?
* `tls.certFile` -> `security.tls.server.certificate.location`
* `tls.trustStore` -> `security.tls.trustStore.location`
* `tls.keyFile` -> `security.tls.server.privateKey.location`
* ``tls.keyPasswordFile` ->  `security.tls.server.privateKey.pwd.location`

You should look at `*Config.java` files in Pravega repo for examples.

Reply:
I think this change needs to happen in the registry service. 
I will make the requisite change and push the update

Reply:
@ravisharda the names used here are just placeholders for taking these values as input from the user. And changing the naming scheme for the charts should not be necessary as the naming is consistent with the charts for our other components.
`tls.secret` and `authentication.passwordAuthSecret` are similar to how we use them in case of our pravega operator.
https://github.com/pravega/pravega-operator/blob/master/doc/tls.md#enable-tls
https://github.com/pravega/pravega-operator/blob/master/doc/auth.md#enable-authentication